test/cluster/test_view_building_coordinator.py

```diff
@@ -79,6 +79,10 @@ async def disable_tablet_load_balancing_on_all_servers(manager: ManagerClient):
     servers = await manager.running_servers()
     await asyncio.gather(*(manager.api.disable_tablet_balancing(s.ip_addr) for s in servers))
 
+async def enable_tablet_load_balancing_on_all_servers(manager: ManagerClient):
+    servers = await manager.running_servers()
+    await asyncio.gather(*(manager.api.enable_tablet_balancing(s.ip_addr) for s in servers))
```

Comment:
tablet balancing is a global config stored in group0, enough to call this using any single host.